src/schema/v2/notifications/Item/PartnerOfferCreatedNotificationItem.ts

```diff
@@ -0,0 +1,53 @@
+import { GraphQLObjectType, GraphQLString } from "graphql"
+import { connectionFromArray } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import { artworkConnection } from "schema/v2/artwork"
+import { createPageCursors } from "schema/v2/fields/pagination"
+import { ResolverContext } from "types/graphql"
+
+export const PartnerOfferCreatedNotificationItemType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "PartnerOfferCreatedNotificationItem",
+  fields: {
+    artworksConnection: {
+      type: artworkConnection.connectionType,
+      args: pageable(),
+      resolve: async (
+        { object_ids },
+        args,
+        { artworksLoader, mePartnerOfferLoader }
+      ) => {
+        if (!mePartnerOfferLoader) return null
+        // object_ids should include a single partner offer id
+        if (object_ids.length === 0) return null
+
+        const { artwork_id } = await mePartnerOfferLoader(object_ids[0])
```

Comment:
Is this right? I think `object_ids` are the artwork id's already, while offer, which isn't needed to resolve this connection, are the `actor_ids`. So you can use `object_ids` directly and don't need this loader here either.